src/python/gem5/utils/simpoint.py

```diff
@@ -94,6 +94,11 @@ def __init__(
             self._warmup_list = self.set_warmup_intervals(warmup_interval)
         else:
             self._warmup_list = [0] * len(self._simpoint_start_insts)
+        
+        if (0 in self._simpoint_start_insts):
+            for index in range(len(self._simpoint_start_insts)):
+                if(self._simpoint_start_insts[index]==0):
+                    self._simpoint_start_insts[index]=1
```

Comment:
Why 1?

Reply:
I worried about scheduling exit events for instruction 0, but scheduling events for instruction 0 is the same as scheduling them for instruction 1. I deleted this part.